.github/workflows/test.yml

```
@@ -49,10 +49,75 @@ jobs:
           DANGER_GITHUB_API_TOKEN: ${{ secrets.DANGER_GITHUB_API_TOKEN }}
         run: bundle exec danger
 
+  ios_test:
+    name: Running tests in iOS
+    runs-on: macos-latest
+    defaults:
+      run:
+        working-directory: ./sample/ios
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
```

please have clear `name` for all steps 🙏 

```suggestion
      - name: Check out
         uses: actions/checkout@v3

      - name: Set up Ruby
         uses: ruby/setup-ruby@v1
```